zproject/urls.py

```diff
@@ -512,6 +512,7 @@
     # export/realm -> zerver.views.realm_export
     rest_path("export/realm", POST=export_realm, GET=get_realm_exports),
     rest_path("export/realm/<int:export_id>", DELETE=delete_realm_export),
+    path("jwt/fetch_api_key", jwt_fetch_api_key),
```

Comment:
I think it makes more sense to add it as rest_path("jwt/fetch_api_ket",POST=jwt_fetch_api_key), no, as the function has require_post decorator on it anyways. Also, can you please remove it from the initial area where it was added directly to urls.py.

Also, I had a talk with a few maintainers and it seems to be the case that the consensus is to push this into the other array rather than this one and rename the array, to avoid the /json/jwt/fetch_api_key endpoint from being auto-generated.

Maybe what you can do this is talk to the maintainers, add it to the other array, rename it after discussing with the maintainers, add the path to the intentionally undocumented list in our openapi tests, and add a pull request for it while we look into the openAPI test issues for this endpoint, so this change can be done independently

Reply:
Thanks, could you please provide a link to the ongoing discussion? Not sure what you mean by  
> the other array

Let me reference the discussion and I can then make the changes accordingly.